dojo/tools/sarif/parser.py

```diff
@@ -302,6 +302,12 @@ def get_item(result, rules, artifacts, run_date):
     if kind != 'fail':
         return None
 
+    # if finding is suppressed, mark it as False Positive
+    # Note: see https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127852
+    suppressed = False
+    if "suppressions" in result and result.get("suppressions") is not None:
```

Comment:
```suggestion
    if result.get("suppressions"):
```
This format is better, it will cover the case where the attribute is here but empty.

>  if the array is empty, a consumer SHALL treat the result as not suppressed.

So if the array is empty a simple if manage it better.

Reply:
Good point. Thank you for being so attentive. I adapted It and outsourced the test case for suppressions. Hope the test are successful.